lib/events.js

```diff
@@ -82,9 +106,10 @@ MapboxEventManager.prototype = {
       return callback();
     }
     var options = this.getRequestOptions(payload);
+    var _this = this;
     this.request(options, function(err,res){
-      if (err) return this.handleError(err, callback);
-      if (res.statusCode != 204) return this.handleError(res, callback);
+      if (err) return _this.handleError(err, callback);
+      if (res.statusCode != 204) return _this.handleError(res, callback);
```

Comment:
Are we able to use arrow functions? If possible let's switch this to an arrow function to avoid the rescope of `this` and avoid having to use `_this`

Reply:
No arrow functions here just now -- perhaps in a future sprint we can bring support for arrow functions. 

In lieu of arrow functions, however, I added a `bind(this)` so we don't need to create a temporary `_this` variable.